modules/tile-converter/src/i3s-converter/helpers/geometry-converter.ts

```diff
@@ -526,38 +530,77 @@ function convertMesh(
         cartographicOrigin,
         cartesianModelMatrix,
         nodeMatrix: matrix,
-        indices: primitive.indices?.value,
+        indices,
         attributeSpecificTransformation: transformVertexNormals,
         useCartesianPositions: false
       })
     );
     outputAttributes.texCoords = concatenateTypedArrays(
       outputAttributes.texCoords,
-      flattenTexCoords(
-        attributes.TEXCOORD_0 && attributes.TEXCOORD_0.value,
-        primitive.indices?.value
-      )
+      flattenTexCoords(attributes.TEXCOORD_0 && attributes.TEXCOORD_0.value, indices)
     );
 
     outputAttributes.colors = concatenateTypedArrays(
       outputAttributes.colors,
-      flattenColors(attributes.COLOR_0, primitive.indices?.value)
+      flattenColors(attributes.COLOR_0, indices)
     );
 
     if (materialUvRegion) {
       outputAttributes.uvRegions = concatenateTypedArrays(
         outputAttributes.uvRegions,
-        createUvRegion(materialUvRegion, primitive.indices?.value)
+        createUvRegion(materialUvRegion, indices)
       );
     }
 
     outputAttributes.featureIndicesGroups = outputAttributes.featureIndicesGroups || [];
     outputAttributes.featureIndicesGroups.push(
-      flattenBatchIds(getBatchIds(attributes, primitive, images), primitive.indices?.value)
+      flattenBatchIds(getBatchIds(attributes, primitive, images), indices)
     );
   }
 }
 
+type TypedArrayConstructor =
```

Comment:
Shall we move this type in some @loaders.gl/core? @ibgreen

Reply:
This should be exported from `@math.gl/types` ? https://github.com/uber-web/math.gl/blob/master/modules/types/src/array-types.ts#L19